doc/releases/release-notes-2.3.rst

```diff
@@ -838,7 +838,7 @@ release:
 * :github:`25369` - tests/drivers/gpio/gpio_basic_api: test_gpio_deprecated step fails on STM32 boards
 * :github:`25366` - tests/drivers/counter/counter_basic_api: instable test status on STM32 boards
 * :github:`25363` - tests/drivers/counter/counter_basic_api: Assertion failed on STM32 boards
-* :github:`25354` - Fails to compile when SYS_PM_DIRECT_FORCE_MODE is true
+* :github:`25354` - Fails to compile when PM_DIRECT_FORCE_MODE is true
```

Comment:
Do we really want to patch (I assume this was done automatically) docs for already done release?

Reply:
no, those will not be part of the change